targets/targets.json

```diff
@@ -15,6 +15,10 @@
         "bootloader_supported": false,
         "static_memory_defines": true,
         "config": {
+            "console-uart": {
+                "help": "Target has UART console on pins STDIO_UART_RTS, STDIO_UART_CTS. Value is only significant if target has SERIAL device.",
```

Comment:
Shouldn't this be stdio_uart_rx and _tx?

Reply:
My specialty was always code, not documentation.